tests/shared/commands.js

```diff
@@ -41,7 +42,13 @@ export const getExample = (id, query = {}, options = {}) => (
     .query(query)
     .set('X-API-Key', options.apiKey || FALLBACK_API_KEY)
 );
-
+export const getWordsFilteredByWordClass = (wordClass, query = {}, options = {}) =>(
+  chai
+  .request(server)
+  .get(`${API_ROUTE}/words/wordClass/${wordClass}`)
+  .query(query)
+  .set('X-API-Key', options.apiKey || FALLBACK_API_KEY)
```

Comment:
These lines should be indented (take a look at another function that uses `chai` as reference)